.gitignore

```diff
@@ -0,0 +1,2 @@
+.env
+__pycache__
```

Comment:
Please add documentation for the necessary environment variables needed so we can run and test this locally.